details/input.md

```diff
@@ -0,0 +1,172 @@
+
+### Lookup table
+
+The site origin lookup table consists of RLE unique site domain values and unique match key provider origins, encoded as 
+ASCII strings. Each section is terminated with a single zero-valued byte.
+
+For example, a query that has two unique site origins and one match key provider will have the lookup table encoded as 
+follows:
+
+```text
+15www.example.com7docs.rs\016matchkeyprovider
+```
+
+All entries are implicitly zero-indexed and the unique index of each entry is used inside the payload to indicate the
+site origin all events within that group are associated with. 
+
+In the example above, `www.example.com` would have index 0 and `docs.rs` would be associated with the index 1. 
+
+### Payload
+
+Payload section carries the match key encryption and event data along with additional authenticated data not included
+in the lookup section. It includes one or more event encoded as follows:
+
+1) The unique index of the site origin for this event encoded as four-byte integer in big-endian byte order. 
+ This index must be unique inside the payload group and be a valid index from the lookup table.
+2) The unique index of the match key provider origin for this event encoded as single-byte integer. 
+3) The single-byte key identifier from the key configuration for the helper party.
+4) The current epoch, encoded as a two-byte integer in big-endian byte order. 
+This index must be unique inside the payload group and be a valid index from the lookup table.
+5) The [HPKE](https://datatracker.ietf.org/doc/html/rfc9180) of replicated match key shares:
+   1) The 32 byte [encapsulated key](https://datatracker.ietf.org/doc/html/rfc9180#section-4) encoded in big endian byte order.
+   2) The PKE of match key shares encoded in big endian order. Using 40 bit match keys will result in 80 bit encryption, etc.
+   3) The 16 byte authentication tag encoded in big endian byte order.
+6) The timestamp of the event encoded as three-byte integer in big-endian byte order. Timestamp of the event is represented as
+number of seconds since the beginning of epoch. 
+7) The secret-shared value of the trigger bit, encoded as two [field](#metadata) values in big-endian order.
+8) The secret-shared value of the trigger value, encoded as two [field](#metadata) values in big-endian order.
+9) The secret-shared value of the breakdown key, encoded as two [field](#metadata) values in big-endian order.
+
+## Simulation
+
+It's worth simulating various distributions of events between unique site origins to estimate potential savings on the wire. 
+Intuitively, the biggest gains can be achieved when relatively few site origin have large number of events associated with them. 
+
+Note: The following estimations ignore the TCP/IP/Ethernet frame overhead as it remains the same regardless of the format chosen
+by the implementations. 
+
+The following simulation assumes each event to take **112 bytes** on the wire, including encryption overhead 
```

Comment:
I think that you need to spell out assumptions here.  Something like:

* enc = 32
* ciphertext = 2*40/8 = 10
* tag = 16
* site = 1 + 50 (say)
* key id = 1
* epoch = 2
* breakdown key = 1 (assuming XOR shares here and a small space, not sure about state of the art)
* trigger value = 4
* ts = 4 (not sure here again)

That's a little more than you have.

But with a table, and if we make breakdown key and trigger value mutually exclusive (and the same size), then we have 68 bytes, plus the table size, which is trivial for a large data set.

The other thing with tables is that you can reuse them...